server/models/alumniModel.ts

```diff
@@ -0,0 +1,31 @@
+import mongoose, { Schema } from "mongoose";
+import { IAlumni } from "../types/alumni";
+
+const alumniSchema = new Schema<IAlumni>(
+  {
+    company: { type: String, trim: true },
+    name: { type: String, trim: true, required: true },
+    email: { type: String, trim: true, required: true },
+    linkedIn: { type: String, trim: true },
+    campus: { type: String, trim: true },
+    cohort: {
+      type: Schema.Types.Mixed, //TODO Better way to do this? Spreadhsete has strings and numbers...
```

Comment:
since the only non number are the PTRI 'beta' maybe we could just call them 0 and bypass the Mixed type. But also we should probably think of a way to validate campus+cohort# combos?

Reply:
Like convert the data as we parse it? That's probably doable but what if Codesmith decides to use that type naming again at some point?